event-checklist.md

```diff
@@ -0,0 +1,104 @@
+* [ ] Sponsorship agreement is signed by sponsors
+* [ ] Post event report is provided to Linux Australia Council
+
+
+##Checklist for Linux Australia
+* [ ] Ensure the event team is aware of the [event subcommittee policy](subcommittee_policy_v2.md)
+* [ ] When received, consider the Motion to form the event subcommittee
+* [ ] Undertake due diligence to ensure that members of the event subcommittee are members of Linux Australia
+* [ ] Provide templated budgets to the event subcommittee to use as the basis for their budget
+* [ ] Review the budget provided by the event subcommittee and provide approval and/or further guidance
+* [ ] Provide access to Westpac banking
+* [ ] Provide access to Xero
+* [ ] Provide a walkthrough of Westpac and Xero
+* [ ] Undertake approvals in Westpac for large transactions in a timely manner
+* [ ] Provide assistance with event infrastructure like website, registration system and payment gateway, if required
+* [ ] Provide an insurance Certificate of Currency if required
+* [ ] Provide templated Sponsorship Agreement if required
+* [ ] Review Sponsorship Agreement if required
+* [ ] Post event report is considered and used as an input into continuous improvement activities
+
+##Forming an event subcommittee
+Forming a high performing team doesn't happen by accident, so ensure that you have the right mix of skills, talent and leadership. It's a good idea to have a clear leader, a 2IC and a Treasurer on your event. Once you have your team together, ensure that they're members of Linux Australia. You must be a member of Linux Australia to sit on a subcommittee, as per the [Linux Australia Constitution](consitution.txt).
+
+A motion is then presented to the Linux Australia Council to form the event subcommittee - see the draft motion below. The Council will consider the motion and will notify you if it has passed, or provide a reason for it being rejected.
+
+###Draft motion
+MOTION by YOUR NAME that an Event Subcommittee is formed to run YOUR EVENT, based on the Subcommittee Policy, with the following members
+* Member - Role
+* Member - Role
+* Member - Role
+
+##Setting up financials
+Once your event subcommittee has formed, Linux Australia will discuss financial arrangements with you and set expectations around the level of profit from your event that will be returned to Linux Australia to cover administrative costs.
+
+You will be provided with access to two systems for financial administration, being;
+* Westpac online banking
+* Xero for accounting and general ledger functions
+
+A Council member or delegate will provide instructions and an overview on how to use the financial systems. In order to gain access to Westpac, you will need to provide the following information to [the Linux Australia Treasurer](mailto:treasurer@linux.org.au);
+
+* Westpac customer number (obtained by providing 100 points of ID at a Westpac branch - passport, driver's licence, Medicare card etc.)

```

Comment:
This isn't clear that the members who need access to Westpac need to go ahead and do this. It reads like the council will provide it.